SECURITY.md

```diff
@@ -0,0 +1,24 @@
+# Security policies
+
+This library intends supporting the development of Solid applications reading and
+writing data in user's Pods. User data should always be considered sensitive and
+be processed carefully, in particular private data which is obtained through
+authentication.
+
+For a better separation of concerns, this library does not deal directly with
+authentication. In order to make authenticated requests, one should inject a `fetch`
+function compatible with the browser-native [fetch API](https://developer.mozilla.org/docs/Web/API/WindowOrWorkerGlobalScope/fetch#parameters)
+dealing with authentication. This may be done using Inrupt's authentication libraries
+[for Node](https://www.npmjs.com/package/@inrupt/solid-client-authn-node) or [for
+the browser](https://www.npmjs.com/package/@inrupt/solid-client-authn-browser).
```

Comment:
Maybe insert a see also to solid client authn’s security policy.

Reply:
fd30e97f57207ac953f289b2530b4131dadf7e52